scripts/test.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import torch
+import torch.nn as nn
+
+cwd = os.getcwd()  # current working directory
+cfp = os.path.dirname(os.path.abspath(__file__))  # current file path
+os.chdir(cfp)
+sys.path.append("..")
+from nn_modules.transformer.top_former import TopFormer
+os.chdir(cwd)
+
+top_former = TopFormer(num_layers=6, d_model=512, nhead=8, num_classes=3)
```

Comment:
Why not add the constants into configuration.

Reply:
To be addressed in a follow-up patch.